plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetViewer.java

```diff
@@ -2944,7 +3061,15 @@ public void run() {
         viewMenu.add(new Separator());
         if (model.getDocumentAttribute() == null) {
             if (valueController != null) {
-                viewMenu.add(new SetRowColorAction(this, attr, valueController.getValue()));
+                String bindingText = attr.getName() 
+                        + " = "
+                        + UITextUtils.getShortText(this.getControl(),
+                        CommonUtils.toString(valueController.getValue()), 100);
+                String msgSelectRowColor = NLS.bind(ResultSetMessages.actions_name_color_by, bindingText);
+                // select row color
+                viewMenu.add(ActionUtils.makeCommandContribution(this.getSite(),
+                        "org.jkiss.dbeaver.core.resultset.grid.selectRowColor",
```

Comment:
Please use a constant you added CMD_SELECT_ROW_COLOR

Reply:
Nice catch